### DISCUSSION TITLE
Not able to build on Vercel, even on clean branch from current working version

### DISCUSSION THREAD

**Discussion by jhughes3818** [vercel/next.js#51817] [Help]
### Summary

I started running into this error when deploying on Vercel

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/vercel/path0/node_modules/axios/lib/helpers/callbackify.js' imported from /vercel/path0/node_modules/axios/lib/adapters/http.js
--
19:06:06.009 | at new NodeError (node:internal/errors:399:5)
19:06:06.009 | at finalizeResolution (node:internal/modules/esm/resolve:326:11)
19:06:06.009 | at moduleResolve (node:internal/modules/esm/resolve:945:10)
19:06:06.010 | at defaultResolve (node:internal/modules/esm/resolve:1153:11)
19:06:06.010 | at nextResolve (node:internal/modules/esm/loader:163:28)
19:06:06.010 | at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
19:06:06.010 | at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
19:06:06.010 | at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
19:06:06.010 | at link (node:internal/modules/esm/module_job:76:36) {
19:06:06.010 | code: 'ERR_MODULE_NOT_FOUND'

I have a current production deployment working fine, and even when I attempt to deploy the same code on a clean branch of main, I get the same error. Any ideas on what might be happening here?

**Comment by icyJoseph** [comment]
What Node version is Vercel running?
